### PR TITLE
fix: color picker a11y fixes

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

- [x] uses `@radix-ui/react-popover` for color picker for a11y reasons
- [x] removed swatch component - had some trouble with the layout while using inside the popover component and it's not used anywhere other than the  color picker component
- [x] use color picker component from `@calcom/ui` in embeds.
- [x] added a border around colors for a small visibility 

before:
Screenshot 2023-02-14 at 13-22-36 Appearance Cal com

after:
Screenshot 2023-02-14 at 13-22-21 Appearance Cal com

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Chore (refactoring code, technical debt, workflow improvements)

### CODE REVIEW

File: apps/web/lib/QueryCell.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ const withQuery = <
   TInput = inferProcedureInput<TQuery>,
   TOutput = inferProcedureOutput<TQuery>
 >(
-  queryProcedure: DecorateProcedure<TQuery, any, any>,
```
Comment (G3root) [inline apps/web/lib/QueryCell.tsx:91]: fixes this error while building

Screenshot 2023-02-14 14:12:07

**Comment by sean-brydon** [comment]
Oh awesome work! Do you thin we could match our DS with this component while youre here 
Not fancied working oh this as i always hated color picker ;') 

https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS?node-id=17036%3A159842&t=o9ntk9G6gBkddtci-4

**Comment by JeroenReumkens** [comment]
Amazing work @G3root  Looks really slick! 
Also like this 🫶
CleanShot 2023-02-14 at 13 42 28@2x

Could you still trigger a deployment to test though? I'd love to test this on an environment 🔥

**Comment by G3root** [comment]
@sean-brydon updated the UI according to the DS 
Screenshot 2023-02-14 at 23-35-23 Appearance Cal com

**Comment by G3root** [comment]
> Amazing work @G3root Looks really slick! Also like this 🫶 CleanShot 2023-02-14 at 13 42 28@2x
> 
> Could you still trigger a deployment to test though? I'd love to test this on an environment fire

all checks have passed  but the preview is not deployed. 🤔

**Review by emrysal — APPROVED** [review]
Verified it works on local, looks good to me, Nice work for negative lines of code 👍

**Pull request merged**